docs/development/modules.md

```diff
@@ -115,17 +114,17 @@ It's recommended to use batch ingestion tasks to validate your implementation.
 The segment will be automatically rolled up to Historical note after ~20 seconds.
 In this way, you can validate both push (at realtime process) and pull (at Historical process) segments.
 
-* DataSegmentPusher
+#### DataSegmentPusher
 
 Wherever your data storage (cloud storage service, distributed file system, etc.) is, you should be able to see one new file: `index.zip` (`partitionNum_index.zip` for HDFS data storage) after your ingestion task ends.
 
-* DataSegmentPuller
+#### DataSegmentPuller
```

Comment:
The extension point is `URIDataPuller` and `DataSegmentPuller` was removed here awhile ago - https://github.com/apache/druid/pull/5461/files
```suggestion
#### URIDataPuller
```